Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM qixxit/elixir-centos
+
+MAINTAINER Ettore Berardi <ettore.berardi@bbc.co.uk>
+
+LABEL uk.co.bbci.api.mozart="true"
```

Comment:
I think we should have this as: uk.co.bbc.news or mozart, but bbci seems unusual.